ddtrace/monkey.py

```diff
@@ -87,7 +87,16 @@
     "gevent": ("gevent",),
     "requests": ("requests",),
     "botocore": ("botocore",),
-    "elasticsearch": ("elasticsearch",),
+    "elasticsearch": (
+        "elasticsearch",
+        "elasticsearch1",
+        "elasticsearch2",
+        "elasticsearch3",
+        "elasticsearch4",
+        "elasticsearch5",
+        "elasticearch6",
```

Comment:
typo I hope
```suggestion
        "elasticesarch6",
```

Reply:
Makes a good case for having tests for each version! 🤦‍♂️

Reply:
done now